test/controllers/FactCheckConfirmControllerSpec.scala

```diff
@@ -42,7 +46,10 @@ class FactCheckConfirmControllerSpec extends ControllerBaseSpec with GuiceOneApp
     val errorHandler: ErrorHandler = injector.instanceOf[ErrorHandler]
 
     val view: fact_check_complete = injector.instanceOf[fact_check_complete]
+    val approvalProcessSummary: ApprovalProcessSummary = ApprovalProcessSummary(id, "title", LocalDate.now, ApprovalStatus.Published)
     val errorView: fact_check_confirm_error = injector.instanceOf[fact_check_confirm_error]
```

Comment:
Personally, I would group the view declarations together.

Reply:
no longer use approvalProcessSummary - so defined together